roles/deploy/tasks/deploy.yml

```diff
@@ -0,0 +1,130 @@
+    dest: "{{ build_path }}"
+    version: "{{ git_version }}"
+    refspec: "+refs/pull/*/merge:refs/remotes/origin/pull-request-*"
+    force: yes
+  tags: clone
+
+- name: symlink into the repo
+  file:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    owner: "{{ unicorn_user }}"
+    state: link
+    force: yes
+  with_items:
+    - { src: "{{ assets_path }}", dest: "{{ build_path }}/public/assets" }
+    - { src: "{{ system_path }}", dest: "{{ build_path }}/public/system" }
+    - { src: "{{ images_path }}", dest: "{{ build_path }}/public/images" }
+    - { src: "{{ spree_path }}", dest: "{{ build_path }}/public/spree" }
+    - { src: "{{ config_path }}/database.yml", dest: "{{ build_path }}/config/database.yml" }
+    - { src: "{{ config_path }}/application.yml", dest: "{{ build_path }}/config/application.yml" }
+    - { src: "{{ l10n_path }}/states.yml", dest: "{{ build_path }}/db/default/spree/states.yml" }
+  tags: symlink
+
+# Install bundler and gems
+
+- name: find current bundler version in Gemfile.lock
+  shell: set -o pipefail && grep -m 1 -A 1 -x -F "BUNDLED WITH" Gemfile.lock | tail -n 1  | tr -d '[:space:]'
+  args:
+    chdir: "{{ build_path }}"
+    executable: /bin/bash
+  changed_when: False
+  register: current_bundler_version
+  tags: bundler
+
+- name: install bundler {{ current_bundler_version.stdout }}
+  gem:
+    name: bundler
+    version: "{{ current_bundler_version.stdout }}"
+    include_doc: no
+  environment: "{{ initialize_rbenv }}"
+  tags: bundler
+
+- name: bundle install
+  bundler:
+    state: present
+    deployment_mode: yes
+    chdir: "{{ build_path }}"
+    gem_path: "/home/{{ unicorn_user }}/.gem"
+  environment:
+    - "{{ initialize_rbenv }}"
+    - "{{ bundler_locale }}"
+  notify:
+    - precompile assets
+    - restart unicorn
+
+# Move new build into place
+
+- name: copy build to current
+  command: cp -TLr {{ build_path }} {{ current_path }}
+  changed_when: True
+
+# Update the database
+
+- name: check database status
+  command: psql -h {{ db_host }} -U {{ db_user }} -d {{ db }} -c "SELECT true FROM pg_tables WHERE tablename = 'order_cycles';"
+  register: table_exists
+  ignore_errors: yes
+  changed_when: False
+  tags: rake
+
+- name: load schema if it isn't loaded
+  command: bash -lc "bundle exec rake db:schema:load RAILS_ENV={{ rails_env }}"
+  args:
+    chdir: "{{ current_path }}"
+  when: table_exists.stdout.find('0 rows') != -1
+  tags:
+    - rake
+    - skip_ansible_lint
+
+- name: migrate database
+  command: bash -lc "bundle exec rake db:migrate RAILS_ENV={{ rails_env }}"
+  args:
+    chdir: "{{ current_path }}"
+  tags:
+    - rake
+    - skip_ansible_lint
+  notify:
+    - precompile assets
+    - restart unicorn
+
+- name: seed database
+  # We run a shell script that passes the default email and password to rake with an EOF block, so we don't hang on the prompts.
+  command: bash -lc "{{ config_path }}/seed.sh RAILS_ENV={{ rails_env }}"
```

Comment:
Aren't we seeding the DB always, including production?

Reply:
Yes, and I think the rake task accepts ENV vars, so we don't need the script.

Reply:
I'll do this in a separate PR :+1: